spec/dlangspec.opf

```diff
@@ -41,11 +41,11 @@
        Additional contributors whose contributions are secondary to those listed in
        creator  elements should be named in contributor elements.
   -->
-	<dc:creator>digitalmars.com</dc:creator>
+	<dc:creator>D Language Foundation</dc:creator>
```

Comment:
Please don't mix unrelated changes in one commit.

Getting meaningful diffs in renamed files depends on the quality of the git implementation's rename detection, so please avoid editing unrelated things in the same commit as renaming the file.

Reply:
I disagree as the reference `git` implementation is smart enough and the main implementation in use.
Anyhow I reverted these changes.

Reply:
1. The *current* version of the reference git implementation is smart enough
2. GitHub does not use the reference git implementation
3. Putting unrelated changes in one git commit is still bad practice.